app/assets/javascripts/active_admin.js

```diff
@@ -71,10 +76,56 @@
     });
   };
 
+  // the 'maximum length' validation in rails automatically adds the 'maxlength' attr on any corresponding page description input, but we want the user to have the freedom to exceed that number
+  function removeMaxLengthAttrFromPageDescription() {
+      $('#page_description').removeAttr('maxLength');
+  }
+
+  function addPageDescriptionCharacterCounterText() {
+      let hintEle = $('#page_description').next();
+      $(hintEle).append(' <span class="page-description-character-counter">(0/140 characters)</span>');
+  }
+
+  function getPageDescriptionCharacterCountOnPageLoad() {
+      let descriptionInput = $('#page_description');
+
+      let characterCounterEle = $('.page-description-character-counter');
```

Comment:
on page load if a description that was previously changed before the changes were introduced had over 140 characters then the text counter isn't red but the valid color
<img width="1649" alt="Screen Shot 2021-11-30 at 1 52 06 PM" src="https://user-images.githubusercontent.com/20211771/144118553-f44e9e81-843c-49f5-99ba-f960bedaa31d.png">

Reply:
Nice catch Elisa. This issue has been resolved!